src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -219,6 +221,65 @@ const ComponentVersionDiff = ({
     rightVersion: rightVersion,
   })
 
+  const HandleExportDiff = () => {
+    const addedResources = []
+    const removedResources = []
+
+    diff.components_changed?.forEach(component => {
+
+      const formatResourceName = (resource, component) => {
+        if (resource.type === 'ociImage') {
+          return `[Docker] ${component.name.split('/', 1)[0]}/${resource.name}`
```

Comment:
This way, you always parse the first part of the component name, e.g. `github.com/gardener/dashboard` -> `github.com`. However, to get the `dashboard` part, you can re-use existing util function `shortenComponentName`.